### PR TITLE
Masking edge case fix

### PR DESCRIPTION
**Pull request by benidis**
The missing value imputations methods break if the window is all `nan` values. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by mbohlkeschneider — COMMENTED** [review]
Thank you! Maybe add a test for that edge case?

**Comment by lostella** [comment]
> Maybe add a test for that edge case?

+1, otherwise looks good, thanks for spotting this!

File: test/test_transform.py [hunk 1]
```diff
@@ -898,6 +898,7 @@ def test_AddObservedIndicator():
     array_values = [
         np.array([np.nan, 1.0, 1.0, np.nan, 2.0, np.nan, 1.0, np.nan]),
         np.array([np.nan]),
```
Comment (lostella) [inline test/test_transform.py:900]: So there was a test with all nan, but it also had just one value!

**Review by lostella — APPROVED** [review]

File: test/test_transform.py (hunk 1 above)
Comment (benidis) [reply test/test_transform.py:900]: Yeah because having a single missing values is another edge case!
